seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/utils/RowConverter.java

```diff
@@ -320,7 +321,7 @@ public static SeaTunnelRow convert(InternalRow rowData, SeaTunnelRowType seaTunn
                     break;
                 default:
                     throw new PaimonConnectorException(
-                            CommonErrorCodeDeprecated.UNSUPPORTED_DATA_TYPE,
+                            CommonErrorCode.UNSUPPORTED_DATA_TYPE,
```

Comment:
Please throw common error by use this method.

https://github.com/apache/seatunnel/blob/a5609d600e0b6da51eab67eee431cef232a2fbb9/seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonError.java#L90

Reply:
> Please throw common error by use this method.
> 
> https://github.com/apache/seatunnel/blob/a5609d600e0b6da51eab67eee431cef232a2fbb9/seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonError.java#L90

done.